easybuild/framework/easyblock.py

```diff
@@ -923,6 +923,10 @@ def make_devel_module(self, create_in_builddir=False):
         WARNING: you cannot unload using $EBDEVELNAME (for now: use module unload `basename $EBDEVELNAME`)
         """
 
+        if not ActiveMNS().mns.det_generate_devel_module():
+            self.log.info("Skipping devel module...")
+            return
```

Comment:
Hmm, we try to avoid inline `return` statements across the codebase (doesn't help with readability).

To avoid changing indentation in this whole method, why not change the condition that guards the (only) call to `make_devel_module` instead to include `and ActiveMNS().mns.det_generate_devel_module()`?